docs/MainSite/Installation.md

```diff
@@ -167,7 +167,7 @@ or (equivalently) use the `module load` command.
 it is recommended you read the [documentation](https://spack.readthedocs.io) if
 you require features such as packages installed with different compilers.
 
-### Building SpECTRE
+### Building SpECTRE (within Spack environment)
```

Comment:
This sub-header is already under the "using Spack" header, so the parenthetical is redundant.

However I am starting to think (and I surmise @haraldp271 does too) that our installation notes page is getting too long, as a result of having both Docker and Spack instructions. This makes it more difficult to navigate within the notes, and means redundant comments like the one above become helpful navigational aids. Perhaps we could instead split the off the Docker and Spack instructions into two sub-pages?

Reply:
We will split docker and spack installation instructions into separate pages in another PR

Reply:
Then let's remove the parenthetical from this PR, as it will not be needed post-split.